packages/react/src/Popover/Popup/PopoverPopup.tsx

```diff
@@ -109,6 +137,40 @@ PopoverPopup.propTypes /* remove-proptypes */ = {
    * Class names applied to the element or a function that returns them based on the component's state.
    */
   className: PropTypes.oneOfType([PropTypes.func, PropTypes.string]),
+  /**
+   * Determines an element to focus after the popover is closed.
+   * If not provided, the focus returns to the trigger.
+   */
+  finalFocus: PropTypes.shape({
+    current: (props, propName) => {
+      if (props[propName] == null) {
+        return null;
+      }
+      if (typeof props[propName] !== 'object' || props[propName].nodeType !== 1) {
+        return new Error(`Expected prop '${propName}' to be of type Element`);
+      }
+      return null;
+    },
+  }),
```

Comment:
```suggestion
  finalFocus: refType,
```

This will also allow you to remove the new entry in overrides/common.json